report/Motivation/Motivation.tex

```diff
@@ -0,0 +1,58 @@
+
+\begin{itemize}
+  \item Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.
+  \item Business people and developers must work together daily throughout the project.
+  \item The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.
+\end{itemize}
+
+These four principles all require collaboration with the customer, we will now reason about the principles and try to suggest what implications they might put on the customer and the developers.
+
+\textbf{Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.} This principle indicates that the developers should deliver vertical slices of the software to the customer as soon as possible and continuously. We assume that this will allow the developers to validate the software early and learn whether the software is correct in terms of actual usage instead of in terms of specified requirements. if our assumption is correct, this would mean that the customer would have to use unfinished or unstable software. Using unfinished software could mean a massive time investment for the customer, since they might have to do everything twice, once in their current system on which their operation still depends upon and once in the new system in order to give feedback to the developers. Using unstable software could not only mean a time investment but also increase the frustration towards the new software. This time investment and frustration could come from the fact that software might behave unexpectable, such as crashing when the customer tries to persist data. The customer than might try to save data multiple times and get frustrating with the software and the process as a whole.
+
+\textbf{Business people and developers must work together daily throughout the project.} This principle has an obvious implication on the customer, the customer has to dedicate time throughout the day to work together with the developers of the project. Given the customer has daily responsibilities regarding their own business one can imagine this to be difficult.
+
+\textbf{The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.} Given the two previous principles, this brings along a rather big implication. Because not only does the customer need to invest a significant time investment into helping the developers, they also need to do it face-to-face, in other terms, they need to be in the same location. This means that when the customer needs to reason about requirements and features they need to do it away from the context and domain of said requirements and features. One can imagine how difficult it must be to reason about requirements and features for a system when alienated from the context and domain of the said system.
```

Comment:
```suggestion
\textbf{The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.}
Given the two previous principles, this brings along a rather big implication.
Because not only does the customer need to invest a significant time into helping the developers, they also need to do it face-to-face, in other terms, they need to be in the same location as the developers. 
This means that when the customer needs to reason about requirements and features they need to do it away from the context and domain of said requirements and features. 
One can imagine how difficult it must be to reason about requirements and features for a system when alienated from the context and domain of the said system.
```